src/Common/AzurePSCmdlet.cs

```diff
@@ -551,6 +522,18 @@ protected new void WriteWarning(string text)
             base.WriteWarning(text);
         }
 
+        protected new void WriteInformation(object messageData, string[] tags)
+        {
+            FlushDebugMessages();
+            base.WriteInformation(messageData, tags);
+        }
+
+        protected void WriteStringInformation(string text, bool? noNewLine = null)
```

Comment:
Having `WriteInformation` and `WriteStringInformation` together is weird. Can you think of any better design?

Reply:
What is the reason of designing a new WriteInformation?

Reply:
It's common for us to write a string to information stream. We need a quick access to write a string to PowerShell console for this instead of a complex usage like 
```
 WriteInformation(new HostInformationMessage { Message = message, NoNewLine =false }, new string[1] { "PSHOST" });
```